Dockerfile

```diff
@@ -28,10 +28,14 @@ RUN apk add --no-cache \
 	git \
 	sqlite
 
+COPY requirements.txt /app/htsget_server/requirements.txt
```

Comment:
This step seems redundant if you already copy over app directory

Reply:
I moved it so that you can keep the pip stuff cached even if there is a code change. pysam takes forever to build.